src/components/InstallTabs/LinuxPanel/index.tsx

```diff
@@ -15,13 +15,13 @@ export const PureLinuxPanel = ({ nvmVersion }: Props): JSX.Element => {
     <div>
       <ShellBox textToCopy={`curl -o- ${nvmInstallScriptUrl} | bash`}>
         <span className="install__text__no-select">$</span>
-        <span className="install__text__command"> curl -o- </span>
```

Comment:
Probably need to move the space to after the `$` so it's out of the "no-select" span, but not having it stuck to the `curl`. Same with line below

Reply:
But the problem is that install__text__no-select adds a whitespace after the $. I don't know why the ci is crashing this because it now does not look any diffrent from for example the macos version of this component.